addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -185,21 +187,35 @@ wTourUtils.registerWebsitePreviewTour('edit_link_popover', {
         run: function () {}, // it's a check
     },
     {
-        content: "Ensure that a click on the link popover link opens a new window",
+        content: "Ensure that a click on the link popover link opens a new window in edit mode",
         trigger: 'iframe .o_edit_menu_popover a.o_we_url_link[target="_blank"]',
         extra_trigger: 'iframe .o_edit_menu_popover a.o_we_full_url[target="_blank"]',
-        run: function (actions) {
-            this.$anchor[0].addEventListener('click', (ev) => {
-                const originalPreventDefault = ev.preventDefault.bind(ev);
-                ev.preventDefault = () => {
-                    throw new Error(
-                        "The link popover should not be default prevented as to open a new tab"
-                    );
-                };
-                originalPreventDefault(); // We do not want to open a new tab in a tour
-            }, {once: true, capture: true});
-            actions.click();
+        run: async function (actions) {
```

Comment:
The async does nothing as we already discussed here https://github.com/odoo/odoo/pull/121264#discussion_r1254226862
The `run()` is not async capable.